flask_login/test_client.py

```diff
@@ -0,0 +1,19 @@
+from flask.testing import FlaskClient
+
+
+class FlaskLoginClient(FlaskClient):
+    """
+    A Flask test client that knows how to log in users
+    using the Flask-Login extension.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
This way is more explicit:
`def __init__(self, user=None, fresh_login=True, *args, **kwargs):`

I know the `FlaskClient` docs show using `kwargs.pop` so either is fine.